nomad/local/start_detach.sh

```diff
@@ -19,7 +19,7 @@ ensure_cros_bridge_networking_workaround() {
         if ! grep -q "_/bridge.ko" "$module_filepath"; then
             echo "It looks like you're on ChromeOS, but haven't installed the Nomad bridge networking workaround."
             # shellcheck source-path=SCRIPTDIR
-            source "$(dirname "${BASH_SOURCE[0]}")/../../etc/chromeos/lib/installs.sh"
+            source "../../etc/chromeos/lib/installs.sh"
```

Comment:
(this was actually failing previously)

Reply:
How was it failing? How was CI (presumably) missing this failure? How does this change fix it?

Reply:
The failure was that the `source` assumed we were in a given directory, but the `cd $THIS_DIR` earlier in the script busts that assumption. The failure looked as follows:

```
wimax@graptop:~/src/grapl$ ./nomad/local/start_detach.sh 
It looks like you're on ChromeOS, but haven't installed the Nomad bridge networking workaround.
./nomad/local/start_detach.sh: line 23: ./nomad/local/../../etc/chromeos/lib/installs.sh: No such file or directory

(note: in this example I executed it manually with a `./nomad/local/start_detach.sh` instead of a `make up-detach`, but it's the same failure)
```

Since the script _already has us in that ./nomad/local directory_, obviously we no longer need to refer to PWD with that prefix.

to answer why CI missed it:
the failure case here happens if:
- you're on chrome OS (so, obviously, already this wouldn't happen in CI ever)
- the `_bridge.ko` file here didn't exist previously

Reply:
🤔 Rather than having assumptions like this baked into individual functions, I think it may be easier to reason about by having the functions assume they're being called relative to the root of the repository, and having any `cd` logic handled at a higher level, as seen here: https://github.com/grapl-security/grapl/blob/58d218b1cd47a374e2374f4bb926e2a66954f16d/bin/graplctl-pulumi.sh#L108-L127

Reply:
(I think I may just revert this change and handle it in another PR.)

Reply:
Sounds good 👍